functional_tests/tests/test_basic.py

```diff
@@ -174,6 +177,55 @@ def test_create_recipe_show_heartbeat(selenium):
     assert len(recipe_list_page.recipe_rows) == 0
 
 
+def test_revision_page_navbar(selenium):
+    """Test the navbar buttons on a recipe revision page."""
```

Comment:
This description is pretty vague. Can it (and the maybe the title) be changed to describe specifically what is being tested?